### PR TITLE
Improve Unit Test Coverage & Update Old References

### PR DESCRIPTION
**Pull request by amsmith-pro**

### CODE REVIEW

File: docs/conops.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 <center>
 
-<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
```
Comment (owlot) [inline docs/conops.md:5]: This seems to be the old style of adding the banner. We don't use <center/> tags anymore. This should be:
```
![Aetheric Banner](https://github.com/aetheric-oss/.github/raw/main/assets/doc-banner.png)
```

File: docs/icd.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <center>
```
Comment (owlot) [inline docs/icd.md]: Same as `conops.md`, use new style:
```
![Aetheric Banner](https://github.com/aetheric-oss/.github/raw/main/assets/doc-banner.png)
```

File: docs/sdd.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 <center>
```
Comment (owlot) [inline docs/sdd.md]: Same as `conops.md`, use new style:
```
![Aetheric Banner](https://github.com/aetheric-oss/.github/raw/main/assets/doc-banner.png)
```

File: README.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png)
```
Comment (owlot) [inline README.md]: use `raw` instead of `blob`

File: client-grpc/README.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png)
```
Comment (owlot) [inline client-grpc/README.md]: use `raw` instead of `blob`

File: client-rest/README.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
-![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)
+![Aetheric Banner](https://github.com/aetheric-oss/.github/blob/main/assets/readme-banner.png)
```
Comment (owlot) [inline client-rest/README.md]: use `raw` instead of `blob`

File: docs/conops.md (hunk 1 above)
Comment (amsmith-pro) [reply docs/conops.md:5]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: docs/icd.md (hunk 2 above)
Comment (amsmith-pro) [reply docs/icd.md]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: docs/sdd.md (hunk 3 above)
Comment (amsmith-pro) [reply docs/sdd.md]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: README.md (hunk 4 above)
Comment (amsmith-pro) [reply README.md]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: client-grpc/README.md (hunk 5 above)
Comment (amsmith-pro) [reply client-grpc/README.md]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: client-rest/README.md (hunk 6 above)
Comment (amsmith-pro) [reply client-rest/README.md]: https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709

File: README.md (hunk 4 above)
Comment (owlot) [reply README.md]: Missed this one

Comment (amsmith-pro) [reply README.md]: ah just didn't get `git add`'d, fixed

**Review by owlot — APPROVED** [review]

**Pull request merged**